script.js

```diff
@@ -1,4 +1,78 @@
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
+var gameMode = "waitingForPlayerName";
```

Comment:
Since it is a string, "waitingForPlayerName" could be spelt normally instead of camelCasing. So it can just be "waiting for player name". 

camelCasing is only used for variable names in javaScript